lib/param_parsing/parameters.go

```diff
@@ -34,6 +35,7 @@ func GetParameters() Params {
 	getopt.StringVarLong(&params.CustomBinaryPath, "bin", 'b', "Custom binary path. Ex: tfswitch -b "+lib.ConvertExecutableExt("/Users/username/bin/terraform"))
 	getopt.StringVarLong(&params.DefaultVersion, "default", 'd', "Default to this version in case no other versions could be detected. Ex: tfswitch --default 1.2.4")
 	getopt.BoolVarLong(&params.HelpFlag, "help", 'h', "Displays help message")
+	getopt.StringVarLong(&params.InstallPath, "install", 'i', "Custom install path. Ex: tfswitch -b "+lib.ConvertExecutableExt("/Users/username"))
```

Comment:
Option description looks a bit messed up with `-b`, doesn't it? 🤔
Also should it probably mention full install path as example for clarity (as in with `installDir` appended)?

Reply:
Will fix/add this now